docs/PSConsoleHostReadline.md

```diff
@@ -0,0 +1,50 @@
+---
+ms.date: 12/07/2018
+schema: 2.0.0
+locale: en-us
+keywords: powershell,cmdlet
+external help file: PSReadLine-help.xml
+title:  PSConsoleHostReadLine
+---
+
+# PSConsoleHostReadLine
+
+## SYNOPSIS
+This function is the main entry point for PSReadLine.
+
+## SYNTAX
+
+```
+PSConsoleHostReadLine
+```
+
+## DESCRIPTION
+
+`PSConsoleHostReadLine` is the main entry point for the PSReadLine module. The PowerShell console
+host automatically loads the PSReadLine modules and calls this function. Under normal operating
```

Comment:
PSReadLine module

Reply:
automatically loads in interactive sessions. The module is not loaded for non-interactive sessions, e.g. `pwsh -File foo.ps1`.

Reply:
I will rework this description and submit an update.